src/test/java/api/loans/DueDateNotRealTimeScheduledNoticesProcessingTests.java

```diff
@@ -435,7 +435,44 @@ public void missingReferencedEntitiesDoNotBlockProcessing() {
         loanPolicyMatcher);
 
     MatcherAssert.assertThat(sentNotices, hasItems(
-      hasEmailNoticeProperties(james.getId(), templateId, noticeToJamesContextMatcher),
-      hasEmailNoticeProperties(steve.getId(), templateId, noticeToSteveContextMatcher)));
+      hasEmailNoticeProperties(james.getId(), TEMPLATE_ID, noticeToJamesContextMatcher),
+      hasEmailNoticeProperties(steve.getId(), TEMPLATE_ID, noticeToSteveContextMatcher)));
+  }
+
+  @Test
+  public void noticeIsDeletedIfReferencedTemplateDoesNotExist() {
+
+
```

Comment:
Please remove these blank lines, I don't think they improve the readability of the test